topics-project/__tests__/signup.test.js

```diff
@@ -0,0 +1,241 @@
+/** @jest-environment jsdom */
+import '@testing-library/jest-dom'
+import { render, screen, fireEvent } from '@testing-library/react'
+import userEvent from '@testing-library/user-event';
+import Signup from '../app/signup/page'
+import { InfoForm } from '../app/signup/_components/infoForm'
+import { MetricsForm } from '../app/signup/_components/metricsForm'
+import * as actions from '../app/signup/actions'
+import { expect } from '@playwright/test';
+
+jest.mock('../app/signup/actions')
+
+// Unit tests for info form component
+describe('Info Form', () => {
```

Comment:
the standard approach is to separate the tests for individual components into independent files. Which would have make this change a bit simpler to understand. Having all the tests in a single file like this makes it difficult to understand test failures.